languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/tagset.txt

```diff
@@ -0,0 +1,105 @@
+
+  TAGS FOR LANGUAGETOOL
+  
+  http://www.languagetool.org/
+  http://community.languagetool.org/
+  http://languagetool.wikidot.com/
+_______________________________________________________________________________
+
+--  NOUNS AND ADJECTIVES  --
+
+    Noun:                   N [gender] [number]
+    Adjective:              J [gender] [number]
+
+                [gender]    m   = masculine
+                            f   = feminine
+                            e   = epicene
+
+                [number]    s   = singular
+                            p   = plural
+                            sp  = singular or plural
+
+    Examples:
+        N m s
+        N f p
+        J e sp
+
+    
+--  MISC  --
+
+    Adverb:                     A
+    Interjection:               I
+    Onomatopeia:                O
+    Cardinal number:            K
+    Abbreviation:               S
+    Proper name:                Z
+    Marker                      M
+    
+    
+--  VERBS  --
+
+    Infinitive:             V inf
+    Conjugation:            V [mood] [tense] [person] [number]
+    Present participle:     V ppr
+    Past participle:        V ppa [gender] [number]
+
+                [mood]      ind     = indicative
+                            con     = conditional
+                            sub     = subjonctive
+                            imp     = imperative
+                            
+                [tense]     pres    = present
+                            psim    = “passé simple”  (past: action done once)
+                            impa    = “imparfait”     (past: action done regularly)
+                            futu    = future
+                
+                [person]    1       = first person
+                            2       = second person
+                            3       = third person
+                
+                [number]    s       = singular
+                            p       = plural
+                            
+    Exceptions:
+        avoir               “V avoir”  instead of  “V”
+        être                “V etre”   instead of  “V”
```

Comment:
yes, I doesn't implement the tagger yet, 
I think that this tagset will be well for arabic,